pydantic/utils.py

```diff
@@ -159,7 +159,7 @@ def sequence_like(v: Any) -> bool:
     return isinstance(v, (list, tuple, set, frozenset, GeneratorType, deque))
 
 
-def validate_field_name(bases: List[Type['BaseModel']], field_name: str) -> None:
+def validate_field_name(bases: Iterable[Type[Any]], field_name: str) -> None:
```

Comment:
Why this change?

Reply:
`validate_field_name` is called from `ModelMetaclass.__new__`, where `bases` is a tuple. This caused an Cython error as in 3.x it is able to understand and check type annotations using `list`. This turns out being not necessary I believe because I disabled the [`annotation_typing` directive](https://cython.readthedocs.io/en/latest/src/tutorial/pure.html#disabling-annotations) as I was getting more errors, but still good to have the annotation fixed anyway.